plugins/notebooklm/skills/notebooklm/SKILL.md

```
@@ -0,0 +1,592 @@
+
+# Mostrar configuração de idioma atual
+notebooklm language get
+
+# Definir idioma para geração de artefatos
+notebooklm language set pt_BR   # Português (Brasil)
+notebooklm language set zh_Hans # Chinês Simplificado
+notebooklm language set en      # Inglês (padrão)
+```
+
+**Códigos de idioma comuns:**
+| Código | Idioma |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)